integration-tests/bridges/sudo-relay.patch

```diff
@@ -1,74 +1,209 @@
+- 	type ValidatorSet = Historical;
+- }
+-
+-+impl pallet_sudo::Config for Runtime {
+-+	type RuntimeEvent = RuntimeEvent;
+-+	type RuntimeCall = RuntimeCall;
+-+	type WeightInfo = ();
+-+}
+-+
+- construct_runtime! {
+- 	pub enum Runtime
+- 	{
+-@@ -1771,6 +1777,9 @@ construct_runtime! {
+-
+- 		// Pallet for migrating Identity to a parachain. To be removed post-migration.
+- 		IdentityMigrator: identity_migrator = 248,
+-+
+-+		// Sudo.
+-+		Sudo: pallet_sudo = 255,
+- 	}
+- }
+-
+-diff --git a/relay/polkadot/Cargo.toml b/relay/polkadot/Cargo.toml
+-index c7f8c92b6..53c87f0e8 100644
+---- a/relay/polkadot/Cargo.toml
+-+++ b/relay/polkadot/Cargo.toml
+-@@ -101,6 +101,8 @@ xcm-builder = { workspace = true }
+-
+- sp-debug-derive = { workspace = true }
+-
+-+pallet-sudo = { workspace = true }
+-+
+- [dev-dependencies]
+- sp-keyring = { workspace = true }
+- sp-trie = { workspace = true }
+-@@ -168,6 +170,7 @@ std = [
+- 	"pallet-staking-runtime-api/std",
+- 	"pallet-staking/std",
+- 	"pallet-state-trie-migration/std",
+-+	"pallet-sudo/std",
+- 	"pallet-timestamp/std",
+- 	"pallet-transaction-payment-rpc-runtime-api/std",
+- 	"pallet-transaction-payment/std",
+-diff --git a/relay/polkadot/src/lib.rs b/relay/polkadot/src/lib.rs
+-index b31948725..39d62fe20 100644
+---- a/relay/polkadot/src/lib.rs
+-+++ b/relay/polkadot/src/lib.rs
+-@@ -1650,6 +1650,12 @@ impl pallet_im_online::Config for Runtime {
+- 	type ValidatorSet = Historical;
+- }
+-
+-+impl pallet_sudo::Config for Runtime {
+-+	type RuntimeEvent = RuntimeEvent;
+-+	type RuntimeCall = RuntimeCall;
+-+	type WeightInfo = ();
+-+}
+-+
+- construct_runtime! {
+- 	pub enum Runtime
+- 	{
+-@@ -1759,6 +1765,9 @@ construct_runtime! {
+- 		// refer to block<N>. See issue #160 for details.
+- 		Mmr: pallet_mmr = 201,
+- 		BeefyMmrLeaf: pallet_beefy_mmr = 202,
+-+
+-+		// Sudo.
+-+		Sudo: pallet_sudo = 255,
+- 	}
+- }
+-
 diff --git a/relay/kusama/Cargo.toml b/relay/kusama/Cargo.toml
-index 9f4601f92..9785cb124 100644
+index 5a600a1d3..4db7ea092 100644
 --- a/relay/kusama/Cargo.toml
 +++ b/relay/kusama/Cargo.toml
-@@ -104,6 +104,8 @@ xcm-builder = { workspace = true }
- 
+@@ -106,6 +106,8 @@ xcm-runtime-apis = { workspace = true }
+
  sp-debug-derive = { workspace = true }
- 
+
 +pallet-sudo = { workspace = true }
 +
  [dev-dependencies]
  sp-keyring = { workspace = true }
  sp-trie = { workspace = true }
-@@ -174,6 +176,7 @@ std = [
+@@ -175,6 +177,7 @@ std = [
  	"pallet-society/std",
  	"pallet-staking-runtime-api/std",
  	"pallet-staking/std",
 +	"pallet-sudo/std",
  	"pallet-timestamp/std",
  	"pallet-transaction-payment-rpc-runtime-api/std",
  	"pallet-transaction-payment/std",
+diff --git a/relay/kusama/src/genesis_config_presets.rs b/relay/kusama/src/genesis_config_presets.rs
+index dda349608..4714ecd51 100644
+--- a/relay/kusama/src/genesis_config_presets.rs
++++ b/relay/kusama/src/genesis_config_presets.rs
+@@ -188,6 +188,9 @@ fn kusama_testnet_genesis(
+			"forceEra": Forcing::NotForcing,
```

Comment:
Not exactly sure if this is the place. After applying the patch (using `git apply integration-tests/bridges/sudo-relay.patch` got the following error:

```
error: corrupt patch at line 192

```

Reply:
hmm, I didn't realize that. It looks like the modified patch file itself was also included in the patch file when I ran `git diff` :D
fixed: https://github.com/polkadot-fellows/runtimes/pull/435/commits/fdc0ef0039afff8f53863965676050676c73b523

Reply:
@pandres95 thank you :)